corehq/apps/app_manager/tests/test_suite_multi_select_case_list.py

```diff
@@ -333,8 +333,7 @@ def assert_form_datums(self, form, datum_id):
     def test_child_of_multiselect(self):
         self.assert_module_datums(self.m0.id, [('instance-datum', 'selected_cases')])
 
-        # datum = selected_cases? Should it be case_id?
-        self.assert_module_datums(self.m1.id, [('datum', 'selected_cases')])
+        self.assert_module_datums(self.m1.id, [('datum', 'case_id')])
```

Comment:
`m1` is a child module right? It should also have a datum from the parent module. Looking at the core navigation code this will still work since it's just looking in the session to see what datums are required and order etc is ignored.

Ultimately however I would expect that child module entries would always have the same datums as the entries from the parent module (+ whatever else they need).

I wrote a test case for this here: https://github.com/dimagi/commcare-hq/pull/new/sk/child-module-bug